src/components/strategies/overview/StrategyFilterSort.tsx

```diff
@@ -18,24 +17,24 @@ export enum EnumStrategySort {
   RoiAscending,
   RoiDescending,
 }
+
 export const strategySortMapping: Record<EnumStrategySort, StrategySort> = {
   [EnumStrategySort.Recent]: 'recent',
   [EnumStrategySort.Old]: 'old',
   [EnumStrategySort.PairAscending]: 'pairAsc',
   [EnumStrategySort.PairDescending]: 'pairDesc',
-  [EnumStrategySort.RoiAscending]: 'roiAsc',
-  [EnumStrategySort.RoiDescending]: 'roiDesc',
+  // default to "trades"
+  [EnumStrategySort.RoiAscending]: 'trades',
+  [EnumStrategySort.RoiDescending]: 'trades',
 };
 const isEnumSort = (
   sort: StrategySort | EnumStrategySort
 ): sort is EnumStrategySort => sort in strategySortMapping;
 
 export const getSortFromLS = (): StrategySort => {
-  const sort = lsService.getItem('strategyOverviewSort');
+  const sort = lsService.getItem('strategyOverviewSort') as any;
   const isRoiSort = sort === 'roiDesc' || sort === 'roiAsc';
-  if (!config.showStrategyRoi && (sort === undefined || isRoiSort))
-    return 'recent';
-  if (sort === undefined) return 'roiDesc';
+  if (sort === undefined || isRoiSort) return 'trades';
```

Comment:
if you go with a migration, then references to roi could be removed from here

Reply:
Do you have an example of LS migration in the code that I can use ?

Reply:
Something like this in localeStorage/index

```typescript
  {
    migrate: (prevFormattedKey) => {
      // Change element you want
      if (prevFormattedKey === `carbon-${NETWORK}-v1.1-strategyOverviewSort`) {
        const prevObj = localStorage.getItem(prevFormattedKey);
        if (!prevObj) return;
        const parsedObj = JSON.parse(prevObj);
        const newObj = parsedObj === 5 ? 4 : parsedObj;
        localStorage.setItem(prevFormattedKey, newObj);
      }
      // If you want to move to a new LS version
      const prefix = `carbon-${NETWORK}-v1.1-`;
      const isMatch = prevFormattedKey.startsWith(prefix);
      if (!isMatch) return;
      const key = prevFormattedKey.slice(prefix.length);
      if (!key) return;
      const nextFormattedKey = ['carbon', NETWORK, 'v1.2', key].join('-');
      migrateAndRemoveItem({ prevFormattedKey, nextFormattedKey });
    },
  },
```